scripts/ninja.js

```diff
@@ -1479,6 +1479,7 @@ function setSortedToString(xs) {
  * @returns {string}
  */
 function getVendorConfigNinja() {
+  if (process.env.ESY === "true") return getEnnvConfigNinja();
```

Comment:
If we're building with `esy` we want to use the same setup as if we had OCaml in our environment